### PR TITLE
docs: improve docs for rolling index creation

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

Improving docs for this command 

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: internal/cli/opsmanager/clusters/indexes_create.go [hunk 1]
```diff
@@ -142,15 +142,33 @@ func (opts *IndexesCreateOpts) indexKeys() ([][]string, error) {
...
+// --sparse sparse
+// --background background.
 func IndexesCreateBuilder() *cobra.Command {
 	opts := &IndexesCreateOpts{}
 	cmd := &cobra.Command{
-		Use:   "create [name]",
+		Use:   "create [indexName]",
 		Short: "Create a rolling index for your MongoDB cluster.",
 		Args:  require.MaximumNArgs(1),
+		Annotations: map[string]string{
```
Comment (JuliaMongo) [inline internal/cli/opsmanager/clusters/indexes_create.go:168]: @gssbzn please add the example tag.

**Review by JuliaMongo — DISMISSED** [review]
LGTM but please do add examples to all .go commands even if they are trivial. In this case, it would be useful to show how to specify a specific field and type of index as a param.

File: internal/cli/opsmanager/clusters/indexes_create.go (hunk 1 above)
Comment (gssbzn) [reply internal/cli/opsmanager/clusters/indexes_create.go:168]: done

**Review by JuliaMongo — APPROVED** [review]

**Pull request merged**